build_tools/compiler/play_processor.rb

```diff
@@ -1,9 +1,28 @@
 require 'erb'
-require 'compiler/template_processor'
+require_relative 'template_processor'
 
 module Compiler
   class PlayProcessor < TemplateProcessor
 
+    @@yield_hash = {
+      layout: '<!-- Page content goes here -->',
+      html_lang: '@htmlLang.getOrElse("en")',
+      page_title: '@title.getOrElse("GOV.UK - The best place to find government services and information")',
+      top_of_page: '@(title: Option[String], bodyClasses: Option[String], htmlLang: Option[String] = None)(head:Html, bodyStart:Html, bodyEnd:Html, insideHeader:Html, afterHeader:Html, footerTop:Html, footerLinks:Html, headerClass:Html = Html.empty, propositionHeader:Html = Html.empty)(content:Html)',
+      head: '@head',
+      body_classes: '@bodyClasses.getOrElse("")',
+      header_class: '@headerClass',
+      proposition_header: '@propositionHeader',
+      content: '@content',
+      body_start: '@bodyStart',
+      body_end: '@bodyEnd',
+      inside_header: '@insideHeader',
+      after_header: '@afterHeader',
+      footer_top: '@footerTop',
+      footer_support_links: '@footerLinks',
+      cookie_message: '@cookieMessage.getOrElse("")',
```

Comment:
This needs to have the HTML fragment that is used as the runtime default in the ERB template, otherwise all Play clients have to supply their own cookie warning.